cmd/habitatctl/commands/ps.go

```diff
@@ -16,6 +16,10 @@ limitations under the License.
 package commands
```

Comment:
wait what is PS again? can we maybe rename this

Reply:
It lists processes that are currently running. I named it to be consistent with linux and docker which both use it